### PR TITLE
Separate hvcf creation

### PR DESCRIPTION
**Pull request by lynnjo**
## Description

_Provide a summary of your changes including motivation and context.
If these changes fix a bug or resolves a feature request, be sure to link to that issue._

## Type of change

_What type of changes does your code introduce? Put an `x` in boxes that apply._

- [ ] `CHANGE` (fix or feature that would cause existing functionality to not work as expected)
- [x] `FEATURE` (non-breaking change which adds functionality)
- [ ] `BUGFIX` (non-breaking change which fixes an issue)
- [ ] `ENHANCEMENT` (non-breaking change which improves existing functionality)
- [ ] `NONE` (if none of the other choices apply. Example, tooling, build system, CI, docs, etc.)

## Checklist:

- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated relevant documentation

## Changelog entry

_Please add a one-line changelog entry below. This will be copied to the changelog file during the release process._
Add functionality to create h.vcf file from PHG created g.vcf file

```release-note

```

### CODE REVIEW

File: src/main/kotlin/net/maizegenetics/phgv2/cli/CreateMafVcf.kt [hunk 1]
```diff
@@ -572,6 +569,8 @@ class CreateMafVcf : CliktCommand(help = "Create g.vcf and h.vcf files from Anch
                         "Checksum=\"Md5\",RefRange=\"${refSeqHash}\">",
                 VCFHeaderVersion.VCF4_2
             )
+        } else {
+           println("convertMetaDataRecordToHVCF: asmHeaders already contains key ${assemblyHaplotypeHash}")
```
Comment (btmonier) [inline src/main/kotlin/net/maizegenetics/phgv2/cli/CreateMafVcf.kt:573]: Should this be added to a logger?

**Review by pjbradbury — APPROVED** [review]

**Review by btmonier — APPROVED** [review]

File: src/main/kotlin/net/maizegenetics/phgv2/cli/HvcfFromPHGGvcf.kt [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+ * This class is used to create h.vcf files from existing PHG created g.vcf files
+ * It only works on g.vcf files that were created by PHG and contain the assembly
+ * and contig information in the header.
+ * The db-path parameter is optional.  If it is not provided, the program will use
+ * the current working directory.  It is expected this folder will contain the AGC
+ * compressed genome data.
+ *
+ * Created h.vcf.gz and h.vcf.gz.csi files will be written to the same folder that contains the gvcf files.
+ */
+class HvcfFromPhgGvcf: CliktCommand(help = "Create  h.vcf files from existing PHG created g.vcf files")  {
```
Comment (tcasstevens) [inline src/main/kotlin/net/maizegenetics/phgv2/cli/HvcfFromPHGGvcf.kt:25]: Does this need to be added to Phg.kt?

**Review by tcasstevens — APPROVED** [review]

**Review by zrm22 — APPROVED** [review]

File: src/main/kotlin/net/maizegenetics/phgv2/cli/HvcfFromPHGGvcf.kt (hunk 2 above)
Comment (lynnjo) [reply src/main/kotlin/net/maizegenetics/phgv2/cli/HvcfFromPHGGvcf.kt]: yes - done.

File: src/main/kotlin/net/maizegenetics/phgv2/cli/CreateMafVcf.kt (hunk 1 above)
Comment (lynnjo) [reply src/main/kotlin/net/maizegenetics/phgv2/cli/CreateMafVcf.kt]: Yes - I created a logger for this class and changed all the println's to myLogger values

**Pull request merged**